libnopegl/src/ngpu/opengl/pipeline_gl.c

```diff
@@ -100,10 +100,12 @@ static GLenum get_gl_indices_type(enum ngpu_format indices_format)
     return gl_indices_type_map[indices_format];
 }
 
-static void bind_vertex_attribs(const struct ngpu_pipeline *s, struct glcontext *gl)
+static void bind_vertex_attribs(const struct ngpu_pipeline *s)
```

Comment:
This looks unrelated to the fence commit

Reply:
Put in a separate commit.